configure.in

```diff
@@ -377,6 +377,8 @@ fi
 AC_SUBST(GSF_FLAGS, [$gsf_flags])
 AC_SUBST(GSF_LIBS, [$gsf_libs])
 
+PKG_CHECK_MODULES([ZLIB], [zlib])
```

Comment:
Not sure if this (and the change to `Makefile.am` below) is the right way to add this dependency.